docs/api-artifacts.rst

```diff
@@ -0,0 +1,200 @@
+.. _api-openeo-extra-artifacts:
+
+====================================
+API: openeo.extra.artifacts
+====================================
+
+.. warning::
+    This is a new experimental API, subject to change.
+
```

Comment:
also add this note:

```

.. versionadded:: 0.44.0

```

(assuming that will be the release it will be added)

Reply:
Squashed current state and added the version + changelog as a separate commit